release/models/policy-forwarding/openconfig-pf-network-instances.yang

```diff
@@ -0,0 +1,133 @@
+submodule openconfig-pf-network-instances {
+  belongs-to openconfig-policy-forwarding {
+    prefix "oc-pf";
+  }
+
+  import openconfig-extensions {
+    prefix "oc-ext";
+  }
+  include openconfig-pf-forwarding-policies;
+
+  organization
+    "OpenConfig working group";
+
+  contact
+    "OpenConfig working group
+     www.openconfig.net";
+
+  description
+    "This submodule contains groupings to attach
+     a policy-forwarding policy to one or more network-instances.
+
+     When attached to a network-instance, the policy-forwarding rules
+     should be evaluated against packets in the specified
+     network-instances.";
```

Comment:
Similar comment to what I left in the previous PR #1150: what are the "packets in the specified network-instances"?

* does that mean that the policy applied to all interfaces associated with the network-instance on ingress (e.g. this change introduces only a new way of configuring the existing functionality), or
* are there any additional expectations w.r.t what packets should be matched (e.g packets redirected from another network-instance via a policy)?

Reply:
The operational use case motivating this PR is for packets ingress to the device which arrive on interfaces (including virtual interfaces) which are mapped to a network-instance.  

I am not aware of an operational use case that we have for packets re-directed from another network-instance.

Reply:
Policy forwarding entries are already defined within a network instance, so what does the enclosing network instance mean?

Reply:
> The operational use case motivating this PR is for packets ingress to the device which arrive on interfaces (including virtual interfaces) which are mapped to a network-instance.

Thus, in terms of functionality, this configuration will be fully equivalent to applying a policy to all interfaces associated with the network-instance? We just adding a "simpler" way to configure this?

Can this be explicitly stated in the description? Thanks

Reply:
Yes, that is correct.  I will make this explicit in the description

Reply:
> Policy forwarding entries are already defined within a network instance, so what does the enclosing network instance mean?

It is true, but without any "apply-policy" configuration, it is ambiguous where the policy is applied?  Further, since OC has a way to apply-policy to interfaces, if none are specified, what should the behavior be?

I'd prefer an explicit leaf for this.   

I will change the name/description from a leaf-ref to network-instance instead to a boolean like 'all-interfaces'.   In our use case, we only want to apply the policy to the same network-instance the policy is defined in, and not some different network-instance.